include/wx/fontutil.h

```diff
@@ -179,6 +179,9 @@ class WXDLLIMPEXP_CORE wxNativeFontInfo
     
     void RealizeResource() const;
 private:
+    // the PostScript names reported in macOS start with a dot for System Fonts, this has to be corrected
+    // otherwise round-trips are not possible, resulting in a Times Fallback
```

Comment:
Just trying to understand: why isn't round trip possible for these fonts? Is it a wx problem or does underlying API handle leading periods specially or something else?

Reply:
macOS issued a warning about this in earlier systems, but in 11.00 makes it fail, defaulting to Times ... that way at least it looks better, although it's not a perfect fix, they want us to use the numeric constants for OS-Fonts